guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -1260,8 +1260,10 @@ WEBVTT
 54:24.455 --> 54:35.483
 [J Jay Couey]: So, instead of speaking for an hour, I already probably spoke too long, I want to be able to answer as many questions as anybody wants to throw at me, even from previous talks.
 
+[Charles Kovess]: Wonderful.
+
 54:38.085 --> 54:38.225
-[J Jay Couey]: of a
+[J Jay Couey]: You're going to find a lot of pushback...I would say try to get Kevin McKernan on here. Try to get Kevin McKernan on here again without me here and 
```

Comment:
Wow, AI missed an entire line. Of course he was speaking a little fast, but still...

Reply:
That surprised me too.